Dockerfile.svc

```diff
@@ -35,13 +35,5 @@ USER shuhitsu
 
 ENV RENKU_SVC_NUM_WORKERS 4
 ENV RENKU_SVC_NUM_THREADS 8
-CMD [ \
-    "sh", "-c", \
-    "gunicorn \
-    renku.service.entrypoint:app \
-    -b 0.0.0.0:8080 \
-    --timeout 600 \
-    --workers $RENKU_SVC_NUM_WORKERS \
-    --worker-class gthread \
-    --threads $RENKU_SVC_NUM_THREADS" \
-]
+
+CMD ["renku", "service", "api"]
```

Comment:
Just a suggestion - if the entrypoint is `["renku", "service"]` then the invocation of the container is simply e.g.

`docker run renku/renku-core api` or `docker run renku/renku-core worker` and in the chart we would just have e.g.

```
args: api
```